data/World/Fire Temple MQ.json

```diff
@@ -79,6 +119,24 @@
             "Fire Temple MQ GS Skull On Fire": "
                 (can_play(Song_of_Time) and can_use(Hookshot) and (has_explosives or logic_rusted_switches)) or
                 can_use(Longshot)",
+            "Fire Temple MQ Narrow Path Room Pot 1": "(Small_Key_Fire_Temple, 3)",
+            "Fire Temple MQ Narrow Path Room Pot 2": "(Small_Key_Fire_Temple, 3)",
```

Comment:
You can get to the Fire MQ Narrow Path room pots on 1 fewer key by falling down from near the Skull on Fire. Note that I put in a PR recently that changes the logic on how you actually get to the Skull on Fire. The fall down to the pots does 1 heart fall damage.